src/privatedns/azext_privatedns/_params.py

```diff
@@ -0,0 +1,84 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+from knack.arguments import CLIArgumentType, ignore_type
+from azure.cli.core.commands.parameters import (get_three_state_flag, tags_type)
+from azext_privatedns._validators import (privatedns_zone_name_type, get_vnet_validator, validate_metadata, validate_privatedns_record_type)
+
+
+def load_arguments(self, _):
+    name_arg_type = CLIArgumentType(options_list=('--name', '-n'), metavar='NAME')
+
+    with self.argument_context('network private-dns') as c:
+        c.argument('tags', tags_type)
+        c.argument('relative_record_set_name', name_arg_type, help='The name of the record set, relative to the name of the Private DNS zone.')
```

Comment:
why is relative record set name defined at root level in CLI?

Reply:
It can be used for any record-set operation under the private-dns cli root.